test/apis/placement/v1/resource_validation_test.go

```diff
@@ -0,0 +1,219 @@
+/*
```

Comment:
rename the file name as api_validation_integration_test.go to keep it consistent with others?

Reply:
please create the tests under test/apis/cluster/v1 to keep it consistent with api path.